R/QC_images.R

```diff
@@ -101,14 +128,14 @@ QC_images = function(sample_meta, raw_counts, annotated_counts, normalized_count
   print("generating cell_lines_present image")
   recovery= annotated_counts %>% dplyr::filter(expected_read, !is.na(CCLE_name)) %>%
     dplyr::select(!any_of(c('members'))) %>%
-    merge(cell_set_meta, by="cell_set", all.x=T) %>%
-    dplyr::mutate(members= ifelse(is.na(members), cell_set, members), # for custom cell sets
-                  expected_num_cl = as.character(members) %>% purrr::map(strsplit, ";") %>% 
+    dplyr::left_join(cell_set_meta, by="cell_set", relationship= 'many-to-one') %>%
+    dplyr::mutate(members= if_else(is.na(members), cell_set, members), # for custom cell sets
+                  expected_num_cl= as.character(members) %>% purrr::map(strsplit, ";") %>% 
                     purrr::map(`[[`, 1) %>% purrr::map(length) %>% as.numeric(),
                   count_type= ifelse(n > count_threshold, 'Detected', 'Low'),
                   count_type= ifelse(n==0, 'Missing', count_type)) %>%
     group_by(profile_id, pcr_plate, pcr_well, count_type, expected_num_cl) %>% 
```

Comment:
These two lines can be replaced with dplyr::count(profile_id, pcr_plate, pcr_well, count_type, expected_num_cl)